proposals/notifications.md

```diff
@@ -0,0 +1,88 @@
+# Solid Notifications
+
+Solid notifications are intended to announce social interactions and data. A Solid notification is an [LDP Resource](https://www.w3.org/TR/ldp/#ldpr) which is submitted to and resides in an [LDP Container](https://www.w3.org/TR/ldp/#ldpc), i.e., a [Solid inbox](inbox.md).
+
+The contents of a notification is application specific. We derive two core principles from this:
+
+1. a notification can contain arbitrary data
+2. a notification is vocabulary agnostic
```

Comment:
So, we don't absolutely know for sure at this time. Discussed this with @rhiaro and we haven't reached a point in which we feel completely comfortable either way. All open for discussion.

Notifications are generally intended to be discovered via inboxes. But, that obviously doesn't imply that all resources are notifications. It might mean that all resources inside an inbox container are notifications. We have the solid-terms:inbox property to discover inboxes, so one can FYN their way to the notifications through ldp:contains thereafter. I suppose the fundamental question is, what about any other resources in the inbox container that are not _notifications_? Should they be in there to begin with? Can I submit a notification with an "attachment", i.e., one resource for the notification itself, and another resource for the accompanying files so to speak. This quickly starts to get tricky if we factor in these possibilities.

The current proposal suggests LDPR as opposed to more specific LDPRS or LDPNR b/c we can see UCs for both. Which means that, we can't necessarily expect solid-terms:Notification on the resource itself, but perhaps only through its ,meta (if that's employed). So, that's certainly possible. The question in this case is, if all resources in the inbox container can be or will be treated as notifications, is it required to type them any way? We can see reasons for both sides.